presto-main/src/test/java/io/prestosql/sql/planner/TestLogicalPlanner.java

```diff
@@ -1526,6 +1527,18 @@ public void testSizeBasedSemiJoin()
                                                 values("T_A"))))));
     }
 
+    @Test
+    public void testExplainAnalyze()
+    {
+        assertPlan("EXPLAIN ANALYZE SELECT regionkey FROM nation",
+                output(
+                        node(ExplainAnalyzeNode.class,
+                                exchange(
+                                        LOCAL,
+                                        GATHER,
+                                        tableScan("nation", ImmutableMap.of("regionkey", "regionkey"))))));
```

Comment:
The assertion passes even if the table scan has more projections than just this one,
so it is a wrong test for the change.

You can see this by changing the query to `EXPLAIN ANALYZE SELECT regionkey, nationkey FROM nation` 
without changing the expected plan. You would want this to fail.

Reply:
I have added a new query